libpod/runtime_pod.go

```diff
@@ -182,3 +183,35 @@ func (r *Runtime) GetRunningPods() ([]*Pod, error) {
 	}
 	return runningPods, nil
 }
+
+// PrunePods removes unused pods and their containers from local storage.
+// If force is give, then running pods are also included in the pruning.
```

Comment:
Wait, what? I don't think pruning running things makes any sense

Reply:
@mheon yeah I dont love this ... just because the cli does it does not mean the API must.  Do we agree to remove it?

Reply:
Agree. Replace with filters IMO, but we can do that later.